### ISSUE TITLE
Click on Fetch All Remotes causes dump if no remotes configured, rather than display a message

### ISSUE THREAD

**Issue by donaldww** [VirtusLab/git-machete-intellij-plugin#762]
## Bug description
Please include steps to reproduce (like `go to...`/`click on...` etc.) + expected and actual behavior.

For more non-trivial issues, we would also appreciate if you included the following details:

### Running environment
 - Git Machete plugin version
 - IDE type [e.g. IntelliJ, PyCharm]
 - IDE version [e.g. 2020.1.4]
 - Operating system [e.g. Windows]

### Logs
Enable logging on debug level for this plugin by adding
```
binding
branchlayout
gitcore
gitmachete.backend
gitmachete.frontend.actions
gitmachete.frontend.graph
gitmachete.frontend.ui
```
to list under `Help` -> `Diagnostic Tools` -> `Debug Log Settings`.<br/>
Then reproduce a bug and attach the logs to the issue.<br/>
To find IntelliJ log file, go to `Help` -> `Show Log in Files`.

Consider placing the logs within the `details` (aka "spoiler") tags as the may be very extensive.

<details>
<summary>Logs</summary>
Logs go here
</details>

### Screenshots
If applicable, add screenshots (or screen recordings, see [Peek](https://github.com/phw/peek#peek---an-animated-gif-recorder) on Linux)
to help explain your problem.

**Comment by PawelLipski** [comment]
Whoops good spot, thanks for reporting! there's #705 but I wasn't aware this causes an error to be thrown. I'll fix in v1.0.1 soon.

Btw out for curiosity, how did you get to know about this tool? :)

**Comment by PawelLipski** [comment]
For the record, the specific error happening as of 2021.1 is:

```
java.lang.IllegalArgumentException: maxThreads must be >=1 but got: 0
	at com.intellij.util.concurrency.BoundedTaskExecutor.<init>(BoundedTaskExecutor.java:55)
	at com.intellij.util.concurrency.BoundedTaskExecutor.<init>(BoundedTaskExecutor.java:41)
	at com.intellij.util.concurrency.AppExecutorUtil.createBoundedApplicationPoolExecutor(AppExecutorUtil.java:76)
	at com.intellij.util.concurrency.AppExecutorUtil.createBoundedApplicationPoolExecutor(AppExecutorUtil.java:61)
	at git4idea.fetch.GitFetchSupportImpl.fetchInParallel(GitFetchSupportImpl.kt:142)
	at git4idea.fetch.GitFetchSupportImpl.access$fetchInParallel(GitFetchSupportImpl.kt:45)
	at git4idea.fetch.GitFetchSupportImpl$fetch$1.invoke(GitFetchSupportImpl.kt:104)
	at git4idea.fetch.GitFetchSupportImpl$fetch$1.invoke(GitFetchSupportImpl.kt:45)
	at git4idea.fetch.GitFetchSupportImpl.withIndicator(GitFetchSupportImpl.kt:211)
	at git4idea.fetch.GitFetchSupportImpl.fetch(GitFetchSupportImpl.kt:101)
	at git4idea.fetch.GitFetchSupportImpl.fetchAllRemotes(GitFetchSupportImpl.kt:87)
	at com.virtuslab.gitmachete.frontend.actions.toolbar.FetchAllRemotesAction$1.run(FetchAllRemotesAction.java:69)
	at com.intellij.openapi.progress.impl.CoreProgressManager.startTask(CoreProgressManager.java:450)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.startTask(ProgressManagerImpl.java:117)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcessWithProgressAsync$5(CoreProgressManager.java:510)
	at com.intellij.openapi.progress.impl.ProgressRunner.lambda$submit$3(ProgressRunner.java:243)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:183)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:705)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:647)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:63)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:170)
	at com.intellij.openapi.progress.impl.ProgressRunner.lambda$submit$4(ProgressRunner.java:243)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1700)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:668)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:665)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1.run(Executors.java:665)
	at java.base/java.lang.Thread.run(Thread.java:829)
```
